src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -1683,6 +1679,114 @@ public void findOneAndReplace() {
     }
   }
 
+  @Nested
+  @Order(8)
+  @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
These seem to be duplicate for `FindOneAndDeleteIntegrationTest`, but just with vector column.
I think I would not add these since it's more code to maintain and does not add much more verification.